src/CloudTest.jl

```diff
@@ -136,7 +136,7 @@ using minio_jll
 import ..Config, ..findOpenPorts, ...AWS, .._cmd, .._wait_for_port
 
 """
-    Minio.with(f; dir, bucket, public, startupDelay, debug)
+    Minio.with(f; dir, bucket, public, startupDelay, debug, waitForPortTimeout)
```

Comment:
Needs adding to the docstring

Reply:
Oh, I had it before but had to undo some changes because I accidentally stripped the trailing white-spaces and did not realize I lost them...